locust/log.py

```diff
@@ -23,6 +23,8 @@ class StdOutWrapper(object):
     Wrapper for stdout
     """
     def write(self, s):
+        if s == None or s.strip() == '':
```

Comment:
Please use `s is None`https://www.python.org/dev/peps/pep-0008/#programming-recommendations